src/views/Voting/CreateProposal/helpers.ts

```diff
@@ -0,0 +1,70 @@
+import { ContextApi } from 'contexts/Localization/types'
+import { format, parseISO, isValid } from 'date-fns'
+import { MINIMUM_CHOICES } from './Choices'
+import { FormState } from './types'
+
+export const combineDateAndTime = (date: Date, time: Date) => {
+  if (!isValid(date) || !isValid(time)) {
+    return null
+  }
+
+  const dateStr = format(date, 'yyyy-MM-dd')
+  const timeStr = format(time, 'HH:mm:ss')
+
+  return parseISO(`${dateStr}T${timeStr}`).getTime() / 1e3
+}
+
+export const getFormErrors = (formData: FormState, t: ContextApi['t']) => {
+  const { name, body, choices, startDate, startTime, endDate, endTime, snapshot } = formData
+  const errors: { [key: string]: string[] } = {}
+
+  if (!name) {
+    errors.name = [t('%field% is required', { field: 'Title' })]
+  }
+
+  if (!body) {
+    errors.body = [t('%field% is required', { field: 'Body' })]
+  }
+
+  if (choices.length < MINIMUM_CHOICES) {
+    errors.choices = [t('Please create a minimum of %num% choices', { num: MINIMUM_CHOICES })]
+  }
+
+  const hasEmptyChoice = choices.some((choice) => !choice.value)
+  if (choices.length === MINIMUM_CHOICES && hasEmptyChoice) {
+    errors.choices = Array.isArray(errors.choices)
+      ? [...errors.choices, t('Choices must not be empty')]
```

Comment:
As this fires the moment you start filling in your first choice - maybe a better error here would be `'Your proposal must include at least 2 choices'`?